setup.py

```diff
@@ -12,5 +12,6 @@
         'google-cloud-firestore',
         'cryptography',
         'flask-json-schema',
-        'beautifulsoup4'],
+        'beautifulsoup4',
+        'pydeepmerge'],
```

Comment:
See my extremely picky and 1000000% optional comment about ending lists with a comma and the terminating character on the next line